lib/appeals/schema/appeals_alert.json

```diff
@@ -20,10 +20,8 @@
     },
     "details": {
       "type": "object",
+      "additionalProperties": true,
       "properties": {
-        "dueDate": {
-          "type": "string"
-        }
```

Comment:
Instead of removing the type altogether, would it be possible to include a multi type for this, ie `type: [string, null]` or `type: [string, object]`? (I'm not sure exactly what you mean by a blank object :) )

Reply:
Empty hash may be more accurate 😄. Sometimes is comes through like:
```json
"alerts": [
  {
    "type": "cavc_option",
    "details": {}
  }
]
```
So not just a key with a null value but completely missing. Their [swagger](https://app.swaggerhub.com/apis/dsva-appeals/appeals-status/2.0.0) defines it that way.

Reply:
Ugh, yea, nevermind then >_<